coilsnake/util/common/project.py

```diff
@@ -25,6 +25,7 @@
     7: "2.2",
     8: "2.3.1",
     9: "3.0"
+	10:"3.33"
```

Comment:
is it necessary to add a new project format version rather than just change the previous line to `9: "3.33"`? did the CoilSnake project format actually change at all since 3.0?

also, if you want to do it this way, you need to also change the line above to `FORMAT_VERSION = 10`